bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "angular-daterangepicker-labworks",
-  "version": "0.2.36",
+  "version": "0.2.37",
```

Comment:
Please change the version of the bootstrap datepicker artifact here too

Reply:
Fixed